cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -63,8 +65,9 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
 
   @Override
   public String getPattern() {
-    return String.format("create dataset instance <%s> <%s> [<%s>]",
```

Comment:
You can give empty string to not specify any properties or description. Since CLI commands are positional, I couldn't think of a better way to do this other than adding new commands.